src/main/kotlin/org/jetbrains/research/ml/codetracker/InitActivity.kt

```diff
@@ -10,11 +12,12 @@ class InitActivity : StartupActivity {
     private val logger: Logger = Logger.getInstance(javaClass)
 
     init {
-        logger.info("${Plugin.PLUGIN_ID}: startup activity")
+        logger.info("${Plugin.PLUGIN_NAME}: startup activity")
```

Comment:
Why did you change PLUGIN_ID into PLUGIN_NAME?

Reply:
Plugin ID is set in plugin.xml and it's `io.github.elena-lyulina.codetracker.codetracker`. Also, there is a `com.intellij.openapi.extensions.PluginId` class which can cause misunderstanding since ours is just a string.

Reply:
Thanks for your explanation!